<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>481</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Tilt Network Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_18" rowstretch="0,1,0" columnstretch="0,0,0">
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="stationInfoLayout"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="COM_gb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>COM config</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="port_cb"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Baud rate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baudrate_cb">
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="FrdmViewerWidget" name="oglw"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="connection_but">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QMenu" name="menuModel">
     <property name="title">
      <string>Model</string>
     </property>
     <addaction name="actionFRDM_K64F"/>
     <addaction name="actionPlane"/>
    </widget>
    <addaction name="actionRefresh_ports"/>
    <addaction name="menuModel"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionSet_size">
   <property name="text">
    <string>Set size</string>
   </property>
  </action>
  <action name="actionReal_time_Filters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Real-time Filters</string>
   </property>
  </action>
  <action name="actionAuto_tab_switch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto tab switch</string>
   </property>
  </action>
  <action name="actionRefresh_ports">
   <property name="text">
    <string>Refresh ports</string>
   </property>
  </action>
  <action name="actionFRDM_K64F">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FRDM K64F</string>
   </property>
  </action>
  <action name="actionPlane">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plane</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrdmViewerWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>src.widgets.frdm_viewer_widget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
